tests/generate_source/main.cpp

```diff
@@ -1,3 +1,20 @@
+/*
+ * Copyright 2018-2019 Arm Limited.
```

Comment:
This file isn't modified in this commit, except adding the copyright header - and hasn't been otherwise changed this year, so we can't really put '2019' in the header :)

Could you leave this change out of the commit please?